functions/src/models/organizations/index.ts

```diff
@@ -0,0 +1,63 @@
+import { FirestoreDataConverter } from '@google-cloud/firestore';
+import { IsArray, IsNotEmpty, IsString } from 'class-validator';
+import { firestore } from 'firebase-admin';
+import DocumentData = firestore.DocumentData;
+
+export enum OrganizationType {
+  healthCare = 'health care',
+  socialServices = 'social services',
+  welfare = 'welfare',
```

Comment:
Let's add an organization type for mutual-aid groups?

```suggestion
  welfare = 'welfare',
  mutualAid = 'mutualAid',
```

Reply:
Also, i'm not really sure of the distinction between these types, are we sure these are the ones we want? Are they clearly defined anywhere?

Reply:
`mututalAid=‘mutual_aid’`

idk either but thinking beyond Covid here in the US it’s called welfare so I imagine there is some distinction. We could probably leverage a social worker or someone to help us with the nomenclature?